mac.js

```diff
@@ -8,11 +8,11 @@ var ncp = require('ncp').ncp
 var series = require('run-series')
 var common = require('./common')
 
-function moveHelpers (frameworksPath, appName, callback) {
-  function rename (basePath, oldName, newName, cb) {
+function rename (basePath, oldName, newName, cb) {
```

Comment:
You'll need to reduce the indentation of the body of this function since you moved it out of `moveHelpers`.